tasks/queue-granules/tests/index.js

```diff
@@ -518,7 +576,7 @@ test.serial('A default concurrency of 3 is used', async (t) => {
 
   await queueGranules(event);
 
-  t.true(pMapSpy.calledOnce);
+  t.true(pMapSpy.calledThrice);
   t.true(pMapSpy.calledWithMatch(
```

Comment:
So `pMap` is called once for mapping over the batches, which there are two of, then once for mapping over the granules in each batch, making three total

In the next assertion though, why are we doing `sinon.match.any` on the first two calls. Shouldn't they also be `sinon.match({ concurrency: 3 })`?